.github/workflows/update-copyright-years-in-license-file.yml

```diff
@@ -0,0 +1,51 @@
+name: Update copyright year(s) in license file
+
+on:
+  schedule:
+    - cron: "0 0 1 * *"
+  workflow_dispatch:
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: FantasticFiasco/action-update-license-year@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This is essentially running code from a repo we don't control with the permission to commit in our repo and probably more.

Sorry, but that's a deal breaker for me security wise. I don't think outdated dates in licenses are a big enough of a deal to justify this risk.

Thanks for the PR though.

Reply:
> This is essentially running code from a repo we don't control with the permission to commit in our repo and probably more.

@byroot hmm I think you could have at least reviewed the workflow first. I do not mean to cause any security risks first of all.

Reply:
It doesn't matter if I review it now, as it could change any time.

Reply:
@byroot ok I see how about the codebase still using unmaintained `stalebot`? What I mean is you cannot control `stalebot` but you still have control over the PR made by this workflow.

Reply:
Stalebot permissions are limited. It can't commit into the repo.

<img width="533" alt="Capture d’écran, le 2022-09-06 à 22 37 47" src="https://user-images.githubusercontent.com/44640/188733460-1dadea74-a5a6-4cc3-9c83-8676e2df1c64.png">

Reply:
> Stalebot permissions are limited. It can't commit into the repo.

@byroot Ok if that's the case. Can you please justify the security risks you mentioned? even though you can still review the PR made by the workflow used.

Reply:
I'm not concerned about the PRs it may open. 

I'm concerned about that action repository being compromised and doing other things with that token, like directly pushing code on the `main` branch.

For such a small help it's really not worth the hassle. I appreciate you trying to contribute to Rails, but there's no point arguing.

Reply:
@byroot Ok point taken. I just wanna know why and not just tell people it will cause security risks by not explaining it to them. We are also trying to contribute to the community in some ways. Also, in the first place, I mean NO harm to the codebase.

Reply:
I explained why in my first comment.